server-example/requirements.txt

```
@@ -0,0 +1,38 @@
+appdirs==1.4.3
+asn1crypto==0.24.0
+click==7.1.1
+configparser==4.0.2
+contextlib2==0.6.0.post1
+cryptography==2.1.4
+distlib==0.3.0
+enum34==1.1.6
+filelock==3.0.12
+Flask==1.1.2
```

I'd write:
```suggestion
Flask>1.1
```
instead. That way we won't be pinned to a particular version forever.